test/CumulativeMerkle.test.js

```diff
@@ -68,6 +68,104 @@ describe('Cumulative Merkle Distribution', function () {
     })
   })
 
+  context('when batch claim array tokens', async function () {
+    let merkleDist
+    let merkleRoot
+    let tokenTotal
+    let proofAccounts
+
+    before(function () {
+      // numRuns must be less or equal to the number of accounts in `cum_dist`
+      const numRuns = Object.keys(dist.claims).length
+      fc.configureGlobal({ numRuns: numRuns, skipEqualValues: true })
+      merkleRoot = dist.merkleRoot
+      tokenTotal = ethers.BigNumber.from(dist.tokenTotal)
+      proofAccounts = Object.keys(dist.claims)
+    })
+
+    beforeEach(async function () {
+      const MerkleDist = await ethers.getContractFactory('CumulativeMerkleDrop')
+      merkleDist = await MerkleDist.deploy(token.address)
+      await merkleDist.setMerkleRoot(merkleRoot)
+      await token.mint(merkleDist.address, tokenTotal)
+    })
+
+    it('should accounts get tokens', async function () {
+      await fc.assert(
+        fc.asyncProperty(
+          fc.integer({ min: 0, max: proofAccounts.length - 1 }),
+          async function (index) {
+            const claimAccounts = proofAccounts.slice((proofAccounts.length / 2) * index, (proofAccounts.length / 2) * (index + 1))
```

Comment:
I don't understand the indexes for the slice. It'd seem like the `claimAccounts` would be empty except for `index = 0` and `index = 1`. E.g., for `proofAccounts.length = 10`:
```
index = 0 --> slice(0, 5)
index = 1 --> slice (5, 10)
index = 2 --> slice (10, 15) --> []
index = 3 --> slice (15, 20) --> []
...
```